README.md

```diff
@@ -0,0 +1,3 @@
+# MT Cars
+
+Filter dataset based on carb number; create bar chart describing mpg data for filtered table
```

Comment:
There's no strict requirement on the format of README.md, it's a good practice to follow structure similar to dash-sample-apps. Sections that helps app users to pick up this app quickly:
- App introduction
- How to use this app (virtual env, how to run this app)
- Specific requirement and how to set them up(optional, some app might have specific requirement before setting it up, like external database dependency)
- App screenshot/screencast(optional)
- File system description(optional, if there's multiple directories inside your code base it'll help)